code/modules/client/preference_setup/loadout/lists/accessories.dm

```diff
@@ -71,7 +71,7 @@
 /datum/gear/accessory/holster
 	display_name = "holster, armpit"
```

Comment:
```suggestion
	display_name = "holster"
```
Need to make it obvious it's not just the armpit one then

Reply:
fair enough, I was just using what was already there